api/specs.proto

```diff
@@ -52,6 +52,12 @@ message NodeSpec {
 	Availability availability = 4;
 }
 
+// NodeAvailability is a wrapper so that specifying availability is optional
+message NodeAvailability {
```

Comment:
This is unnecessary. If we wanted to do that, we should have chosen a meaningless zero-value.